### PR TITLE
Make vessel buildable on MacOS

### PR DESCRIPTION
**Pull request by luc-blaeser**
Resolving a build error on MacOS (missing Security library).

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -8,7 +8,11 @@ let
   vessel = rustPlatform.buildRustPackage rec {
     pname = "vessel";
     version = "0.6.5";
-    buildInputs = [ openssl_3_0 openssl_3_0.dev ];
+    buildInputs = [ 
+      openssl_3_0 openssl_3_0.dev 
+      ] ++ pkgs.lib.optionals pkgs.stdenv.isDarwin [
+        pkgs.darwin.apple_sdk.frameworks.Security
+      ];
```
Comment (ggreif) [inline default.nix:15]:
```suggestion
      ] ++ pkgs.lib.optional pkgs.stdenv.isDarwin
        pkgs.darwin.apple_sdk.frameworks.Security;
```

Comment (ggreif) [reply default.nix:15]: Thanks for this @luc-blaeser! If MacOS failed in the past, that implies we didn't have a CI job for it?

**Comment by luc-blaeser** [comment]
Thank you, Gabor. I am not sure whether the macos build runs on the normal CI on pull requests, maybe on release workflow only...

**Review by rvanasa — APPROVED** [review]
Looks good to me! For some reason I'm still missing write permissions for Vessel (despite being the maintainer), so @ggreif might have to give you approval to merge.

**Review by ggreif — APPROVED** [review]
LGTM!

**Pull request merged**